test/nbft/diffs/NBFT-dhcp-ipv6

```diff
@@ -21,7 +21,7 @@ hfi_list[0]->tcp_info.host_name=nvmeof-sles
 hfi_list[0]->tcp_info.this_hfi_is_default_route=1
 hfi_list[0]->tcp_info.dhcp_override=1
 subsystem_ns_list[0]->index=1
-subsystem_ns_list[0]->num_hfis=1
+subsystem_ns_list[0]->num_hfis=2
 subsystem_ns_list[0]->hfis[0]->index=1
 subsystem_ns_list[0]->hfis[1]->index=1
```

Comment:
Does it make sense for `hfis[0]` and `hfis[1]` to have the same value? AFAICS the spec doesn't explicitly say it's illegal. The wording is: "*If multiple HFIs are associated with this record, subsequent interfaces should be populated in the Secondary HFI Associations field*". Because a HFI index identifies an interface uniquely, it makes no sense to me to list the same interface multiple times. It has the potential to confuse consumers of these values.

Same comment multiple times below.